app/models/user.rb

```diff
@@ -83,7 +86,7 @@ def profile_edited?
     if email == "" || email.nil?
       return false
     elsif example_url1.nil? || example_url1_img.nil? || example_url2.nil? || example_url2_img.nil? || techinterests.nil? ||
-       location.nil? || skill.nil?
+       location.nil?
```

Comment:
Align the operands of a condition in a `elsif` statement spanning multiple lines.